### ISSUE TITLE
user.css in protostar is ignored on 404

### ISSUE THREAD

**Issue by Giuse69** [joomla/joomla-cms#25286]
### Steps to reproduce the issue
Go to an invalid link that throws a 404 error

### Expected result
user.css should be included

### Actual result
just template.css is included but not user.css

### System information (as much as possible)
Joomla 3.9.8

**Comment by Quy** [comment]
Cannot reproduce.  `user.css` is in the same folder as `template.css`.

Make sure it is in the following location:
`\templates\protostar\css\user.css`

**Comment by Giuse69** [comment]
Yes, it is. user.css is normally included across the site (also visible in the HTML source of the page) but when I type a wrong URL the source contains


< style>
h1, h2, h3, h4, h5, h6, .site-title {
 font-family: 'Open Sans', sans-serif;
 }
< /style>
< link href="/templates/protostar/css/template.css" rel="stylesheet" />
< link href="/templates/protostar/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
< style>
.......
}

so only template.css is referenced (the blank after < is added by me otherwise the code is not shown here in the message). You get something else?

**Comment by Quy** [comment]
From the page source:
```
<link href="/templates/protostar/css/template.css?31ffb900ec04f4fda475b2d2fcf5e941" rel="stylesheet" />
<link href="/templates/protostar/css/user.css?31ffb900ec04f4fda475b2d2fcf5e941" rel="stylesheet" />
```

I don't know what you mean by typing the wrong URL.

**Comment by franz-wohlkoenig** [comment]
> I don't know what you mean by typing the wrong URL.

so he get a 404-Page where he is missing the user.css

**Comment by Giuse69** [comment]
No, sorry, I mean: I explicitly invoke a non existing URL to receive a 404. In that case protostar template uses template.css but  not user.css

**Comment by Quy** [comment]
In Administration, go to `Extensions > Templates > Templates`.
Click `Protostar Details and Files`.
Click `css` folder icon.
Do you see `user.css` file?

**Comment by Quy** [comment]
I understand now. On the 404 error page, `user.css` is not included.

**Comment by Quy** [comment]
Why not create a custom 404 error page?
https://docs.joomla.org/Creating_a_Custom_404_Error_Page

**Comment by Giuse69** [comment]
Yes, it's an option but I think it should be automatically done. Maybe just adding in "templates/protostar/error.php" around line 70 the inclusion of user.css if existing?

**Comment by Quy** [comment]
It is a generic error page. What kind of custom styling do you need?

**Comment by Giuse69** [comment]
for example to tune representation of the site logo, that is part of the generic error page

**Comment by brianteeman** [comment]
It's a bug. It should be used

**Comment by Giuse69** [comment]
thanks

**Comment by ghost** [comment]
Closed as having Pull Request #25289

**Comment by ghost** [comment]
Reopened as Pull Request #25289 is closed

**Comment by ghost** [comment]
Closed as having Pull Request #25416
